src/assertions/descendantsCount.js

```diff
@@ -0,0 +1,22 @@
+export default function descendantsCount ({ wrapper, markup, arg1, arg2=1, sig }) {
+  this.assert(
+      wrapper.hasDescendantsCount(arg1, arg2),
+      () => 'expected ' + sig + ' to have ' + arg2 + ' descendants #{exp} ' + markup(),
+      () => 'expected ' + sig + ' not to have ' + arg2 + ' descendants #{exp} ' + markup(),
+      arg1
+  )
+}
+
+function convenienceHelper(count) {
+  return function ({ wrapper, markup, arg1, sig }) {
+    const arg2 = count;
+    descendantsCount.call(this, { wrapper, markup, arg1, arg2, sig});
+  };
+}
+
+export const descendantsCountWrappers = {
```

Comment:
"BDD" can easily be overused in test matchers - personally I'd prefer to err on the side of caution (also i'd expect `to.have.descendantsCount()`, fwiw)